### PR TITLE
Functional components/layout/* with css modules

### PR DESCRIPTION
**Pull request by amassarwi**
~~waiting on #3215 as it conflicts with `layout.less`.~~

This refactors `components/layout/*` components to functional approach and splits `layout.less` to **many** local, component specific, encapsulated css modules. 

It also adds few className props to custom style parts of `TabbedPage` & `TabbedPageTab`, before this commit we used to override classes in too many different `less` files but with this approach wrapper will provide specific className defined in it's css module instead of overriding children css classes in wrapper's `.less` file.

This also deletes couple of redux connectors & `MyTickets.less` which wasn't used anymore.

Part of #2438 & #2439.

### CODE REVIEW

File: app/hooks/useModalVisible.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useSelector } from "react-redux";
+import * as sel from "selectors";
+
+const useModalVisible = () => {
```
Comment (bgptr) [inline app/hooks/useModalVisible.js:4]: This custom hook is not used anywhere.

**Review by bgptr — COMMENTED** [review]
LGTM! Left a tiny comment. Nice work!

File: app/hooks/useModalVisible.js (hunk 1 above)
Comment (amassarwi) [reply app/hooks/useModalVisible.js:4]: It's used in `BlurableContainer.jsx`:

https://github.com/decred/decrediton/pull/3278/files#diff-24b84bdbeaf013c28b29eec0c133f6dae393ad8231a1caccdb62928b18d70229R6

Comment (bgptr) [reply app/hooks/useModalVisible.js:4]: Yes, but it imports its own custom hook from `./hooks` and not from `hooks`

Comment (amassarwi) [reply app/hooks/useModalVisible.js:4]: hrmmmm

Comment (amassarwi) [reply app/hooks/useModalVisible.js:4]: You are 💯 right, thanks for this nice catch!

**Review by matheusd — APPROVED** [review]
Tested fine

**Review by alexlyp — APPROVED** [review]

**Pull request merged**